recipes/cmappy/meta.yaml

```diff
@@ -0,0 +1,51 @@
+package:
+  name: cmappy
+  version: "1.0.9"
+
+source:
+  fn: cmapPy-1.0.9.tar.gz
+  url: https://pypi.python.org/packages/91/8d/fa4f925ce1ab47b95e091ff595e905b959bd8f628578e5b10898db275a39/cmapPy-1.0.9.tar.gz
+  md5: 27cabdf06940d60592dabd5850a8f25b
+
+
+build:
+  skip: True
```

Comment:
This will now skip every build, so you will not get any package. What do you wanted to do?